national_holidays.gemspec

```diff
@@ -12,9 +12,9 @@ Gem::Specification.new do |s|
   s.license     = 'MIT'
   s.files       = Dir.glob("{bin,lib,test}/**/*") + Dir.glob("national-holidays-config/conf/*/*.yml") + %w(LICENSE.txt README.md Rakefile)
 
-  s.required_ruby_version = '>= 3.1'
+  s.required_ruby_version = '>= 3.4'
 
-  s.add_runtime_dependency 'countries', '~>7.0'
+  s.add_runtime_dependency 'countries', '~> 8.0'
```

Comment:
Maybe should drop the version requirements/suggestions from the holidays gemspec completely? Or change them to `>=`?

Examples to be guided by:
* https://github.com/countries/countries/blob/master/countries.gemspec
* https://github.com/faker-ruby/faker/blob/main/faker.gemspec
* https://github.com/rails/rails/blob/main/activerecord/activerecord.gemspec

Reply:
Yeah exactly. We already version the countries gem in the Charlie app's Gemfile.lock so it shouldn't upgrade unexpectedly, it should just stop us having to update the gem as well in cases like this.

That said, in this case we should update the gem because of the test failure and fix you had to make, so maybe it's good that we were forced to do that.

Reply:
Recently there's been a major update for the countries gem 1-2 times a year, so although it's a bit of a faff it doesn't happen too often. I would argue it's also quite useful to have the opportunity to familiarise ourselves on how to update and publish a gem too 😅
I think for now I'll keep the restriction, but bear it in mind and if it feels like it's getting too annoying a task we could revisit the decision?